Aztec/Classes/Converters/HTMLNodeToNSAttributedString.swift

```diff
@@ -113,13 +113,12 @@ class HMTLNodeToNSAttributedString: SafeConverter {
         let childAttributes = attributes(forNode: node, inheritingAttributes: inheritedAttributes)
 
         for child in node.children {
-            let childContent = convert(child, inheritingAttributes: childAttributes)
-
+            let childContent = NSMutableAttributedString(attributedString:convert(child, inheritingAttributes: childAttributes))
```

Comment:
I tried reverting this change and everything works fine.  Any reason why you're converting the string to be mutable here?